apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -748,8 +752,17 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
         element, "feedback", FeedbackType, FEEDBACK_DEFAULT
     )
     answer_weight = pl.get_integer_attrib(element, "weight", WEIGHT_DEFAULT)
+
+    # For backward compatibility, we need to override the default partial credit type
+    # when grading_method = ORDERED
+    default_partial_credit = (
+        PartialCreditType.NONE
+        if grading_method is GradingMethodType.ORDERED
+        else PartialCreditType.LCS
+    )
+
```

Comment:
Looks good to me

Reply:
For documentation sake in case someone in the far future dig up this PR, this line is necessary because the existing behaviour for `partial-credits` defaults to `LCS` for all grading methods EXCEPT for`grading-method=ordered`, which defaults to `none`. This line is necessary to maintain this behaviour and maintain backward compatibility and not break existing PL questions which were created before partial credit was created for `ordered`.